.circleci/src/workflows/native_promotion.yml

```diff
@@ -1,10 +1,7 @@
-triggers:
-  - schedule:
-      cron: '0 2 * * 1' # For sanity checks for this syntax: https://crontab.guru/
-      filters:
-        branches:
-          only:
-            - main
+when:
```

Comment:
The branch restriction is now defined in the Triggers and not in the code any more, right?

Reply:
Yes, correct. See https://circleci.com/docs/2.0/scheduled-pipelines/#migrate-scheduled-workflows